AGENTS.md

```diff
@@ -68,6 +67,17 @@ Do not substitute other tools or commands.
 
 ---
 
+## Moving and Deleting Files
+
+- When moving, renaming, or deleting files, ensure all internal references are updated accordingly.
+- When moving files within this repository, use `git mv` to preserve file history.
+- Create redirects for any moved, renamed, or deleted files to maintain link integrity.
+- Redirects should be added to the S3 redirects files located in `/scripts/redirects/`.
```

Comment:
I've never touched this, so I might need some education here. Based on a quick look, I think this only applies when we're moving things that are not in Hugo?